scripts/generate-changelog.js

```diff
@@ -51,8 +51,7 @@ const main = async () => {
   const releaseDate = process.argv[2];
   const from = process.argv[3];
   const to = process.argv[4];
-  const searchQuery = `repo:gitpod-io/gitpod is:pr is:merged merged:${from}..${to} sort:updated-desc`;
-
+  const searchQuery = `repo:gitpod-io/gitpod is:pr is:merged merged:${from}..${to} sort:updated-desc label:deployed -label:release-note-none`;
```

Comment:
```suggestion
  const searchQuery = `repo:gitpod-io/gitpod is:pr is:merged merged:${from}..${to} sort:updated-desc label:deployed`;
```

I think we don't need this @filiptronicek? 

It [looks like the script](https://github.com/gitpod-io/website/blob/main/scripts/generate-changelog.js#L101) already respects the release notes field.

Reply:
@loujaybee this change leads to the PRs with no release note field just being ignored when initially fetching the list, so less PRs have to be processed and it should take way less time to query and create the changelog.

Reply:
Ah, I see this makes the filter below:

```javascript
.filter((identity) => identity)
```

Theoretically redundant? I'm happy to keep it as a safeguard, though. 

I guess we also do need to do the release note parsing since we pull the text.

Reply:
Oh my, it for sure makes the `hasReleaseNote` check redundant, but.... I have no idea what the filter you mentioned does 😝. 

Looks like it should only get rid of [falsy](https://developer.mozilla.org/en-US/docs/Glossary/Falsy) items, but I don't see how these could happen (the API does not return empty records). Seems like nothing has changed when removing it, so let's just do that.

Reply:
[`74ba5c2` (#2759)](https://github.com/gitpod-io/website/pull/2759/commits/74ba5c2983dc9799d632fe6f8034a254c36a0675)